python/pyspark/sql/connect/client.py

```diff
@@ -967,9 +967,8 @@ def _execute_and_fetch_as_iterator(
                                 "streaming_query_command_result": b.streaming_query_command_result
                             }
                         if b.HasField("streaming_query_manager_command_result"):
-                            yield {
-                                "streaming_query_manager_command_result": b.streaming_query_manager_command_result
-                            }
+                            cmd_result = b.streaming_query_manager_command_result
```

Comment:
If I don't do this, `dev/reformat-python` always changes this line to
```
yield {
                                "streaming_query_manager_command_result": b.streaming_query_manager_command_result
                            }
```
which would result to python lint error because the second line has length 115

Reply:
Looks fine to me.